### PR TITLE
Update the 'Chat with other developers' section #876

### PR DESCRIPTION
**Pull request by yash-nisar**
Update the 'Chat with other developers' section to include crucial
links of the official Gitter channel and a mailing list for the
ease of communication.

Signed-off-by: Yash Nisar <yash.nisar@somaiya.edu>

### CODE REVIEW

**Comment by yash-nisar** [comment]
Wanna review @haikoschol @pombredanne ?

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -38,9 +38,18 @@ If you are proposing a feature:
...
 ==========================
 
-You can join the official #aboutcode IRC channel on freenode
-(server chat.freenode.net) to chat about scancode and other related tools.
-You can use your favorite IRC client or use the web chat at https://webchat.freenode.net/
+For other questions, discussions, and chats, we have:
+
+- a mailing list at https://lists.sourceforge.net/lists/listinfo/aboutcode-discuss
+
+- an official Gitter channel at https://gitter.im/aboutcode-org/discuss
```
Comment (pombredanne) [inline CONTRIBUTING.rst]: I would put Gitter first, and de-emphasize IRC and the mailing list IMHO.

**Comment by pombredanne** [comment]
Looking good. See my comment

File: CONTRIBUTING.rst (hunk 1 above)
Comment (yash-nisar) [reply CONTRIBUTING.rst]: I picked it up from https://github.com/nexB/scancode-toolkit#support to maintain consistency 😛 . No problem, I'll change it in a minute 😉

**Comment by pombredanne** [comment]
Thanks! merging

**Pull request merged**